tests/test_expected_protocol.py

```diff
@@ -33,16 +35,56 @@ class BasicTestInstrument(Instrument):
     )
 
 
+class InstrumentWithPreprocess(BasicTestInstrument):
+    def values(self, command, **kwargs):
+        return super().values(command, preprocess_reply=lambda v: v+"2345", **kwargs)
+
+    def __init__2(self, adapter, **kwargs):
```

Comment:
Oops, that 2 should not be there I think.

Reply:
No, the 2 deactivates the init, depending how we further decide to impement stuff.
If we decide to not support (in tests) an adapter preprocessor, we can drop this init altogether.

Reply:
I'll fix this.